src/components/CategoryPage.js

```diff
@@ -9,9 +9,9 @@ const CategoryPage = ({ activities, title, overviewText, importanceText, keySkil
             container
             direction="column"
             justifyContent="center"
-            alignItems="center"
+            alignItems="left"
             spacing={2}
-            p={8}
+            p={6}
```

Comment:
Let's change this back to p={8} since that spacing looks better. Instead to address the extra spaces, add in conditional rendering of the importanceText and keySkillsText Grids only when  those values are not empty.